Traceback (most recent call last):
  File "C:\Users\wschroder\AppData\Roaming\Python\Python311\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\Lib\site-packages\nbclient\client.py", line 1263, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\Lib\site-packages\nbclient\util.py", line 85, in wrapped
    return just_run(coro(*args, **kwargs))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\Lib\site-packages\nbclient\util.py", line 60, in just_run
    return loop.run_until_complete(coro)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\Lib\asyncio\base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\Lib\site-packages\nbclient\client.py", line 701, in async_execute
    await self.async_execute_cell(
  File "C:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\Lib\site-packages\nbclient\client.py", line 1019, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\Lib\site-packages\nbclient\client.py", line 913, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from matplotlib import rcParams, cycler
import matplotlib.pyplot as plt
import numpy as np
plt.ion()
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mImportError[0m                               Traceback (most recent call last)
[1;32m~\AppData\Local\Temp\ipykernel_21308\2191868119.py[0m in [0;36m<cell line: 0>[1;34m()[0m
[1;32m----> 1[1;33m [1;32mfrom[0m [0mmatplotlib[0m [1;32mimport[0m [0mrcParams[0m[1;33m,[0m [0mcycler[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m      2[0m [1;32mimport[0m [0mmatplotlib[0m[1;33m.[0m[0mpyplot[0m [1;32mas[0m [0mplt[0m[1;33m[0m[1;33m[0m[0m
[0;32m      3[0m [1;32mimport[0m [0mnumpy[0m [1;32mas[0m [0mnp[0m[1;33m[0m[1;33m[0m[0m
[0;32m      4[0m [0mplt[0m[1;33m.[0m[0mion[0m[1;33m([0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m

[1;32mC:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\Lib\site-packages\matplotlib\__init__.py[0m in [0;36m<module>[1;34m[0m
[0;32m    111[0m [1;31m# cbook must import matplotlib only within function[0m[1;33m[0m[1;33m[0m[0m
[0;32m    112[0m [1;31m# definitions, so it is safe to import from it here.[0m[1;33m[0m[1;33m[0m[0m
[1;32m--> 113[1;33m [1;32mfrom[0m [1;33m.[0m [1;32mimport[0m [0m_api[0m[1;33m,[0m [0m_version[0m[1;33m,[0m [0mcbook[0m[1;33m,[0m [0m_docstring[0m[1;33m,[0m [0mrcsetup[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m    114[0m [1;32mfrom[0m [0mmatplotlib[0m[1;33m.[0m[0mcbook[0m [1;32mimport[0m [0msanitize_sequence[0m[1;33m[0m[1;33m[0m[0m
[0;32m    115[0m [1;32mfrom[0m [0mmatplotlib[0m[1;33m.[0m[0m_api[0m [1;32mimport[0m [0mMatplotlibDeprecationWarning[0m[1;33m[0m[1;33m[0m[0m

[1;32mC:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\Lib\site-packages\matplotlib\rcsetup.py[0m in [0;36m<module>[1;34m[0m
[0;32m     25[0m [1;32mfrom[0m [0mmatplotlib[0m [1;32mimport[0m [0m_api[0m[1;33m,[0m [0mcbook[0m[1;33m[0m[1;33m[0m[0m
[0;32m     26[0m [1;32mfrom[0m [0mmatplotlib[0m[1;33m.[0m[0mcbook[0m [1;32mimport[0m [0mls_mapper[0m[1;33m[0m[1;33m[0m[0m
[1;32m---> 27[1;33m [1;32mfrom[0m [0mmatplotlib[0m[1;33m.[0m[0mcolors[0m [1;32mimport[0m [0mColormap[0m[1;33m,[0m [0mis_color_like[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m     28[0m [1;32mfrom[0m [0mmatplotlib[0m[1;33m.[0m[0m_fontconfig_pattern[0m [1;32mimport[0m [0mparse_fontconfig_pattern[0m[1;33m[0m[1;33m[0m[0m
[0;32m     29[0m [1;32mfrom[0m [0mmatplotlib[0m[1;33m.[0m[0m_enums[0m [1;32mimport[0m [0mJoinStyle[0m[1;33m,[0m [0mCapStyle[0m[1;33m[0m[1;33m[0m[0m

[1;32mC:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\Lib\site-packages\matplotlib\colors.py[0m in [0;36m<module>[1;34m[0m
[0;32m     54[0m [1;32mimport[0m [0mmatplotlib[0m [1;32mas[0m [0mmpl[0m[1;33m[0m[1;33m[0m[0m
[0;32m     55[0m [1;32mimport[0m [0mnumpy[0m [1;32mas[0m [0mnp[0m[1;33m[0m[1;33m[0m[0m
[1;32m---> 56[1;33m [1;32mfrom[0m [0mmatplotlib[0m [1;32mimport[0m [0m_api[0m[1;33m,[0m [0m_cm[0m[1;33m,[0m [0mcbook[0m[1;33m,[0m [0mscale[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m     57[0m [1;32mfrom[0m [1;33m.[0m[0m_color_data[0m [1;32mimport[0m [0mBASE_COLORS[0m[1;33m,[0m [0mTABLEAU_COLORS[0m[1;33m,[0m [0mCSS4_COLORS[0m[1;33m,[0m [0mXKCD_COLORS[0m[1;33m[0m[1;33m[0m[0m
[0;32m     58[0m [1;33m[0m[0m

[1;32mC:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\Lib\site-packages\matplotlib\scale.py[0m in [0;36m<module>[1;34m[0m
[0;32m     20[0m [1;32mimport[0m [0mmatplotlib[0m [1;32mas[0m [0mmpl[0m[1;33m[0m[1;33m[0m[0m
[0;32m     21[0m [1;32mfrom[0m [0mmatplotlib[0m [1;32mimport[0m [0m_api[0m[1;33m,[0m [0m_docstring[0m[1;33m[0m[1;33m[0m[0m
[1;32m---> 22[1;33m from matplotlib.ticker import (
[0m[0;32m     23[0m     [0mNullFormatter[0m[1;33m,[0m [0mScalarFormatter[0m[1;33m,[0m [0mLogFormatterSciNotation[0m[1;33m,[0m [0mLogitFormatter[0m[1;33m,[0m[1;33m[0m[1;33m[0m[0m
[0;32m     24[0m     [0mNullLocator[0m[1;33m,[0m [0mLogLocator[0m[1;33m,[0m [0mAutoLocator[0m[1;33m,[0m [0mAutoMinorLocator[0m[1;33m,[0m[1;33m[0m[1;33m[0m[0m

[1;32mC:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\Lib\site-packages\matplotlib\ticker.py[0m in [0;36m<module>[1;34m[0m
[0;32m    136[0m [1;32mimport[0m [0mmatplotlib[0m [1;32mas[0m [0mmpl[0m[1;33m[0m[1;33m[0m[0m
[0;32m    137[0m [1;32mfrom[0m [0mmatplotlib[0m [1;32mimport[0m [0m_api[0m[1;33m,[0m [0mcbook[0m[1;33m[0m[1;33m[0m[0m
[1;32m--> 138[1;33m [1;32mfrom[0m [0mmatplotlib[0m [1;32mimport[0m [0mtransforms[0m [1;32mas[0m [0mmtransforms[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m    139[0m [1;33m[0m[0m
[0;32m    140[0m [0m_log[0m [1;33m=[0m [0mlogging[0m[1;33m.[0m[0mgetLogger[0m[1;33m([0m[0m__name__[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m

[1;32mC:\Program Files\ArcGIS\Pro\bin\Python\envs\arcgispro-py3\Lib\site-packages\matplotlib\transforms.py[0m in [0;36m<module>[1;34m[0m
[0;32m     47[0m [1;33m[0m[0m
[0;32m     48[0m [1;32mfrom[0m [0mmatplotlib[0m [1;32mimport[0m [0m_api[0m[1;33m[0m[1;33m[0m[0m
[1;32m---> 49[1;33m from matplotlib._path import (
[0m[0;32m     50[0m     affine_transform, count_bboxes_overlapping_bbox, update_path_extents)
[0;32m     51[0m [1;32mfrom[0m [1;33m.[0m[0mpath[0m [1;32mimport[0m [0mPath[0m[1;33m[0m[1;33m[0m[0m

[1;31mImportError[0m: numpy.core.multiarray failed to import
ImportError: numpy.core.multiarray failed to import

